packages/console/src/hooks/use-user-preferences.ts

```diff
@@ -11,7 +11,7 @@ import { themeStorageKey } from '@/consts';
 import useApi, { RequestError } from './use-api';
 
 const userPreferencesGuard = z.object({
-  language: z.nativeEnum(Language).optional(),
+  language: z.enum(languageKeys).optional(),
```

Comment:
maybe use languageKeyGuard.optional() which has already been defined in @logto/shared?

Reply:
It will failed